fenix/app/src/main/java/org/mozilla/fenix/home/topsites/TopSites.kt

```diff
@@ -315,6 +319,14 @@ private fun TopSiteItem(
             }
         }
     }
+
+    val context = LocalContext.current
+    val activity = context as? HomeActivity
+    LaunchedEffect(Unit) {
```

Comment:
I don't know too much about side effects in Compose, but I am wondering if using `Unit` here might be too relaxed? Would `onTopSitesItemBound` be called more than necessary if there's a recomposition after the home screen is fully drawn? This might not be an actual problem, but I just want to make sure we understand this well.

I am trying to also determine if calling `reportFullyDrawn` multiple times has any consequences. I want to assume it will just log the first time it is called and the subsequent calls will do nothing, but want to double check my understanding.

Reply:
`LaunchedEffect(Unit)` will only be called at first composition

Reply:
> `LaunchedEffect(Unit)` will only be called at first composition

Perfect!